packages/frontend/core/src/components/workspace/index.tsx

```diff
@@ -33,25 +34,26 @@ export const AppContainer = ({
 };
 
 export interface MainContainerProps extends HTMLAttributes<HTMLDivElement> {
-  className?: string;
-  padding?: boolean;
-  transparent?: boolean;
+  clientBorder?: boolean;
 }
 
 export const MainContainer = forwardRef<
   HTMLDivElement,
   PropsWithChildren<MainContainerProps>
 >(function MainContainer(
-  { className, padding, children, transparent, ...props },
+  { className, children, clientBorder, ...props },
   ref
 ): ReactElement {
+  const appSideBarOpen = useAtomValue(appSidebarOpenAtom);
   return (
     <div
       {...props}
       className={clsx(mainContainerStyle, className)}
       data-is-macos={environment.isDesktop && environment.isMacOs}
-      data-show-padding={!!padding}
-      data-transparent={transparent}
+      data-show-padding={!!clientBorder}
+      data-transparent={false}
```

Comment:
I guess this & the css rule can also be removed

Reply:
yes, removed